bum/song.py

```diff
@@ -9,12 +9,12 @@
 from . import util
 
 
-def init():
+def init(port=6600):
     """Initialize mpd."""
     client = mpd.MPDClient()
 
     try:
-        client.connect("localhost", 6600)
+        client.connect("localhost", f"{port}")
```

Comment:
Just `port` can be used here. Or if that doesn't work: `str(port)`.

Reply:
Yes jut <code> port </code> worked.